### ISSUE TITLE
[ISSUE] Provider bug: Error: resource databricks_notebook doesn't support import

### ISSUE THREAD

**Issue by evaimitico** [databrickslabs/terraform-provider-databricks#404]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Databricks Provider issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.13.2
+ provider registry.terraform.io/databrickslabs/databricks v0.2.8
+ provider registry.terraform.io/hashicorp/azurerm v2.35.0

### Affected Resource(s)
- databricks_notebook

### Environment variable names
DATABRICKS_HOST
DATABRICKS_TOKEN

### Debug Output
```
[0m[1mmodule.mydatabricks.databricks_notebook.notebook["Shared/01-simsearch-indexing/01-phase1/01-es2sentences.py"]: Importing from ID "/Shared/01-simsearch-indexing/01-phase1/01-es2sentences"...[0m
2020/11/09 09:54:19 [ERROR] eval: *terraform.EvalImportState, err: resource databricks_notebook doesn't support import
2020/11/09 09:54:19 [ERROR] eval: *terraform.EvalSequence, err: resource databricks_notebook doesn't support import
2020/11/09 09:54:19 [TRACE] [walkImport] Exiting eval tree: module.mydatabricks.databricks_notebook.notebook["Shared/01-simsearch-indexing/01-phase1/01-es2sentences.py"] (import id "/Shared/01-simsearch-indexing/01-phase1/01-es2sentences")
2020/11/09 09:54:19 [TRACE] vertex "module.mydatabricks.databricks_notebook.notebook[\"Shared/01-simsearch-indexing/01-phase1/01-es2sentences.py\"] (import id \"/Shared/01-simsearch-indexing/01-phase1/01-es2sentences\")": visit complete
2020/11/09 09:54:19 [TRACE] No provider meta schema returned
2020/11/09 09:54:19 [TRACE] GRPCProvider: Configure
```

### Expected Behavior
when running 

```
terraform import 'module.mydatabricks.databricks_notebook.notebook["Shared/01-sdh-simsearch-indexing/01-phase1/01-es2sentences.py"]' "/Shared/01-sdh-simsearch-indexing/01-phase1/01-es2sentences"
```

i expect the existing notebook to be imported in terraform state.

### Actual Behavior
the command ends with:
Error: resource databricks_notebook doesn't support import

is this functionality implemented as stated in documention?

Thank you

**Comment by nfx** [comment]
@evaimitico ack, will add in 0.2.9 bugfix release.

Just in case, import should be by `notebook_id`, not by `notebook_path`, as you're specifying here.

**Comment by evaimitico** [comment]
Hi nfx,
thanks for adding to 0.2.9
At first i tried with notebook_id but since it didn't work i tried with path too.
I'll wait for 0.2.9!

**Comment by nfx** [comment]
Will be fixed in #415
